src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests.cs

```diff
@@ -5221,27 +5221,24 @@ partial class C
 
             var compilation = CreateCompilationWithMscorlib(source, null, options: opt);
             compilation.VerifyDiagnostics(
-                // (25,25): error CS0759: No defining declaration found for implementing declaration of partial method 'C.Foo6<T>()'
-                //     static partial void Foo6<[A][A] T>() { }
-                Diagnostic(ErrorCode.ERR_PartialMethodMustHaveLatent, "Foo6").WithArguments("C.Foo6<T>()"),
```

Comment:
This error should not be dropped. Restored in https://github.com/dotnet/roslyn/pull/10359.